iroh-net/Cargo.toml

```diff
@@ -47,7 +47,7 @@ num_enum = "0.7"
 once_cell = "1.18.0"
 parking_lot = "0.12.1"
 pin-project = "1"
-pkarr = { version = "1.1.4", default-features = false, features = ["async", "relay"] }
+pkarr = { version = "2.0.0", default-features = false, features = ["async", "rand", "relay"] }
```

Comment:
updating was necessary to convert from [u8; 32] to PublicKey. adding "rand" was necessary because the crate doesn't build otherwise

Reply:
Can you open an issue at the pkarr repo? We should not have to fix the crate by adding features...

Reply:
opened a PR: https://github.com/Nuhvi/pkarr/pull/66